.github/workflows/stable-tag.yml

```diff
@@ -1,24 +1,16 @@
-name:  Tag with Stable
+name: Tag with Stable
```

Comment:
Does the revised versioning workflow not already apply a stable tag in its tagging step?

Reply:
i.e. is there a reason to have this workflow at all?

Reply:
outside if we need to run this one manually on some occaion, we don't need it.
In the `git-playground` repo, I actually deactivated it...

Reply:
actually, I'm ready to delete it: I can't see any scenario where we would have to run this: I expect the release workflow to be more reliable for this now.